NuGet.config

```diff
@@ -3,7 +3,8 @@
   <packageSources>
     <clear />
     <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
-    <add key="darc-pub-dotnet-corefx-8a3ffed" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-corefx-8a3ffed5/nuget/v3/index.json" />
+    <add key="darc-pub-dotnet-core-setup-76619ef" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-core-setup-76619ef8/nuget/v3/index.json" />
+    <add key="darc-int-dotnet-corefx-36b8b8e" value="https://pkgs.dev.azure.com/dnceng/_packaging/darc-int-dotnet-corefx-36b8b8e2/nuget/v3/index.json" />
```

Comment:
@wtgodbe @mmitche @silvioguiso why is Maestro++ adding internal isolated feeds in dependency updates❔

Reply:
It's probalby that the locations didn't get updated yet. Once they are on nuget.org, the locations in the MAestro DB are updated nightly. If they are on nuget.org (or other general purpose feed like dotnet3.1, then the update PR doesn't get this).

You can probably kick it by closing and re-triggering this update.

Reply:
/fyi @riarenas

Reply:
Yep, I did just that and https://github.com/dotnet/extensions/pull/3402 looks how I expected